src/common/SPELLS_DEMON_HUNTER.js

```diff
@@ -68,10 +89,14 @@ export default {
   IMMOLATION_AURA: {
     id: 178740,
     firstStrikeSpellId: 187727,
```

Comment:
I hadn't seen this before, but I'd much prefer having separate spells than adding new properties. This keeps the SPELLS object super consistent making it easier to use, as well as requires you to give the spells proper names (e.g. what's now the `id` could be called `IMMOLATION_AURA_CAST`, if that's what it is anyway).